File: backend/scripts/create_admin.py
import requests
import json

def create_admin():
    url = 'http://localhost:3000/register'
    data = {
        'name': 'Admin',
        'email': 'admin@besustainable.com',
        'password': '1234'
    }
    
    try:
        response = requests.post(url, json=data)
        print('Status:', response.status_code)
        print('Response:', response.json())
    except Exception as e:
        print('Error:', str(e))

if __name__ == '__main__':
    create_admin() 

File: backend/scripts/create-admin.js
const axios = require('axios');

const createAdmin = async () => {
  try {
    const response = await axios.post('http://localhost:3000/register', {
      name: 'Admin',
      email: 'admin@besustainable.com',
      password: '1234'
    });
    console.log('Admin criado com sucesso:', response.data);
  } catch (error) {
    console.error('Erro ao criar admin:', error.response?.data || error.message);
  }
};

createAdmin(); 